test/test_other.py

```diff
@@ -160,7 +160,7 @@ def assertIsWasmDylib(self, filename):
     self.assertTrue(building.is_wasm_dylib(filename))
 
   def do_other_test(self, testname, emcc_args=None, **kwargs):
-    self.do_run_in_out_file_test('other', testname, emcc_args=emcc_args, **kwargs)
+    return self.do_run_in_out_file_test('other', testname, emcc_args=emcc_args, **kwargs)
```

Comment:
What is the return value we need to propagate here?

Reply:
It's the output of running the test, which is needed to extract the profile.